pyserini/encode/__main__.py

```diff
@@ -91,6 +91,9 @@ def parse_args(parser, commands):
                               required=True)
     input_parser.add_argument('--fields', help='fields that contents in jsonl has (in order)',
                               nargs='+', default=['text'], required=False)
+    input_parser.add_argument('--docid-field',
+                              help='name of id field name. `default` would be either of `id`, `_id`, `docid`',
+                              default=None, required=False)
```

Comment:
@ehsk, Overall looks good to me. Maybe here you can add by default it is `id`. This way someone is not confused in the future with the `--docid-field`, i.e. why does it work with `None`.

Also in the helper we can mention that the `doc_id` can be any key (few common options are `id`, `_id` or `docid` .